themes/vocabulary_theme/templates/blog-post.html

```diff
@@ -6,55 +6,70 @@
 
 {% set author = this.parent.parent.children.get('authors').children.get(this.author) %}
 
-{% block title %}{{ this.title }}{% endblock %}
-{% block header %}{{ this.parent.title }}{% endblock %}
 {% block body %}
-  <h2 class="mb-0">{{ this.title }}</h2>
-  {{ check_file('content' + this.parent.parent.path + '/authors/' + this.author + '/contents.lr') }}
-  <p class="meta text-muted mt-0 mb-2">by <a href="{{ author|url }}">{{ render_author_name(author) }}</a>
-  on {{ this.pub_date|dateformat('EEEE, yyyy-''MM-d') }}</p>
-  {% if this.series|length %}  
-   {% set series = this.parent.parent.children.get('series').children.get(this.series) %}
-  <div class="card bg-light mb-2">
-    <div class="card-body p-2">
-    <p class="meta my-0"><span class="mr-2"><strong>This blog is part of the series:</strong></span>
-       <a href="#series">{{ series.name }}</a>
-    </p>
+  <div class="single-entry">
+    <div class="single-entry-header">
```

Comment:
because of the website semantic, this header div should be a `<header>`  element